Node.js/basicresponse.js

```diff
@@ -221,7 +228,7 @@ function respondToRequest(hConn, hObj, msgObject, mqmdRequest) {
       // Describe how the Put should behave
       pmo.Options = MQC.MQPMO_SYNCPOINT;
 
-      //If you do not disconnect, the default is a commit; if you do not disconnect and commit then the default is a rollback
+      // If any error is detected in the put operation, the message will rollback; else it will be commited
```

Comment:
In the comment clarify that it is the original gotten request that is being rolled back onto the queue.

Reply:
Still not quite right with the test. Change it to - 


// If any error is detected in the reply put operation, then we have not been able to handle 
// the original request received. That message needs to be rolled back onto the queue. 
// If there is no error, then both our read of the request and our send of the reply can be 
// committed. That will permanently take the request off the queue, and commit the reply 
// allowing it to be read by the requesting application.